projects/urllib3/build.sh

```diff
@@ -15,7 +15,7 @@
 #
 ################################################################################
 
-python3 setup.py install
+pip install .
```

Comment:
```suggestion
python3 -m pip install .
```
I expect this should fix the two failing tests - https://github.com/google/oss-fuzz/runs/4990624155#step:8:156
I also think it's better practice to invoke pip through Python - https://stackoverflow.com/a/67084664/4093019.

Reply:
Thanks, I should have done that from the start indeed